src/portal/azext_portal/generated/_params.py

```diff
@@ -0,0 +1,46 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from argcomplete.completers import FilesCompleter
+from azure.cli.core.commands.parameters import (
+    tags_type,
+    resource_group_name_type,
+    get_location_type,
+    file_type
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('portal dashboard list') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The name of the resource group.')
+
+    with self.argument_context('portal dashboard show') as c:
+        c.argument('resource_group_name', resource_group_name_type, help='The name of the resource group.')
+        c.argument('name', help='The name of the dashboard.')
```

Comment:
Please change the argument name to the full `dashboard_name`, but add `options_list=['--name', '-n']`.

Reply:
Why you prefer `dashboard_name`?
I think 1)"name" is cleaner and clearer since every cmd followed `az portal dashboard XXX`, the name param is definitely for the portal dashboard. 2)For consistency consideration, I notice other extensions are using `--name` like storage, VM, container registry, etc.

Reply:
Using `options_list=['--name', '-n']` also allows you to use `--name` as the option plus the more convenient short option `-n`, but for argument name, there are cases that you could have more than three resource names in the same command. To make the names clear during development and maintenance, the convention is to use the full name for argument.

Reply:
I renamed the parameter though the configuration in readme.az.md file in azure-rest-api-specs repo. That's why the argument name also changed to `name`. But it looks cmd params can be renamed in `option-list` as well. I'm confused about which one is better to use.
@fengzhou-msft @RodgeFu Any suggestion?